jsonparser.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <sstream>      // std::istringstream
+#include <map> 
+#include <fstream>
+
+class jsonparser{
+private:
+public:
+    static std::string rFVbQ (std::string st){   // alias: returnFirstValuebetweenQuotationmarks, finds the first substd::string that sits between " marks and returns it
+        if (st.find('"') == std::string::npos) throw "invalid json syntax"; // if doesn't contain throw
+        std::string tmp = st.substr(st.find('"')+1);
+        if (tmp.find('"') == std::string::npos) throw "invalid json syntax"; // if doesn't contain throw
```

Comment:
Kicsit gyorsulna, ha a find visszaadott értékét elmentenéd ahelyett, hogy újra hívnád a függvényt. Meg persze elég lenne egy substr hívás is, a két find által visszaadott érték közti részre, de ezek apróságok.

Reply:
A find-ok számából sikerült egyet le faragni, azonban a második substr-től nem tudtam megszabadulni mert máshogy nem tudom a második ' " '-et megtalálni.